src/components/common/ui/progress-mui/Progress.styles.ts

```diff
@@ -0,0 +1,14 @@
+import { SxProps, Theme } from '@mui/material/styles';
+
+export const progressFront: SxProps<Theme> = {
+  position: 'relative',
+  '.MuiCircularProgress-svg': {
+    strokeLinecap: 'round',
+  },
+};
+
+export const progressBack: SxProps<Theme> = {
+  color: 'backgroundDark.300',
+  position: 'absolute',
+  left: 0,
```

Comment:
Does it do smth? 
<img width="119" alt="Screenshot 2023-05-27 at 22 31 05" src="https://github.com/fictadvisor/fictadvisor-web/assets/89708817/67408910-e738-4e15-8ad0-5b8e8c12e741">
<img width="244" alt="Screenshot 2023-05-27 at 22 31 08" src="https://github.com/fictadvisor/fictadvisor-web/assets/89708817/96d5600b-5e2e-4562-b818-d99f7f0e7529">